plugins/modules/system/open_iscsi.py

```diff
@@ -57,6 +57,11 @@
         - Whether the target node should be automatically connected at startup.
         type: bool
         aliases: [ automatic ]
+    auto_portal_startup:
+        description:
+        - Whether the target node portal should be automatically connected at startup.
+        type: bool
+        aliases: [ automatic_portal ]
```

Comment:
I would avoid adding new aliases if not strictly necessary.
```suggestion
        version_added: 3.2.0
```

Reply:
i thought of resembling this with `automatic` that is meant for target and `automatic_portal` to point specific portal of the target.

Reply:
about version_added i have included in commit _580e1af_

Reply:
removed alias completeley in commit _f9973c5_